server/actions_apis.go

```diff
@@ -178,7 +179,11 @@ func (s *RegistryServer) ListApis(ctx context.Context, req *rpc.ListApisRequest)
 			if err != nil {
 				return nil, invalidArgumentError(err)
 			}
-			if !out.Value().(bool) {
+			v, ok := out.Value().(bool)
+			if !ok {
+				return nil, invalidArgumentError(errors.New("expression does not evaluate to a boolean"))
```

Comment:
Can we offer any information about the expression in this error message? Would printing out `prg` provide any useful information for understanding the error?

Reply:
Good idea, I've added the expression's type to the error.